src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -94,23 +126,33 @@ public LocalDateTime getDateInSpecificTimeZone(String dateInString, String zone)
      * OffsetDateTime is recommended to use for storing date values in a database.
      */
     public OffsetDateTime offsetDateTime(LocalDateTime localTime) {
-        return OffsetDateTime.now();
+        return OffsetDateTime.of(localTime, ZoneOffset.of("+2"));
     }
 
     /**
      * Given a String formatted as `yyyymmdd`,
      * return Optional of this date as a LocalDate.
      */
     public Optional<LocalDate> parseDate(String date) {
-        return Optional.empty();
+        try {
+            return Optional.of(LocalDate.parse(date,
+                    DateTimeFormatter.ofPattern(YEAR_MONTH_DAY_NOSPACES)));
```

Comment:
we can use standard pattern here DateTimeFormatter.BASIC_ISO_DATE

Reply:
Is there any other like this one for two other methods? I found `DateTimeFormatter.RFC_1123_DATE_TIME` but it's not suitable.